packaging/targets/trusty/debian/install

```diff
@@ -0,0 +1 @@
+/usr/share/nginx/html/*
```

Comment:
+1 for nginx :) Maybe it'd be better to put the console somewhere else though, in case the implementer is already using nginx and have existing content in there?

What we could do is add a entry for the console in `/etc/nginx/sites-available/` pointing to another location (maybe `/usr/share/openhim-console`?), then enable this in site-enabled. What do you think?